frontend/src/router/index.js

```diff
@@ -55,6 +56,11 @@ const router = new Router({
       name: "past",
       component : PastAssignment,
     },
+    {
```

Comment:
Remove these changes to resolve conflicts.

Reply:
Resolved